custom_components/tuya_local/devices/henden_essential_pool_heatpump.yaml

```diff
@@ -0,0 +1,146 @@
+name: Pool heat pump
+products:
+  - id: 4t4vo1voussnybbd
+    manufacturer: Henden
+    model: Essential Inverter Heat Pump
+entities:
+  - entity: climate
+    translation_only_key: pool_heatpump
+    dps:
+      - id: 101
+        name: hvac_mode
+        type: boolean
+        mapping:
+          - dps_val: false
+            icon: "mdi:hvac-off"
+            icon_priority: 1
+            value: "off"
+          - dps_val: true
+            constraint: mode
+            conditions:
+              - dps_val: "warm"
+                value: heat
+              - dps_val: "cool"
+                value: cool
+                available: cool_available
+              - dps_val: "smart"
+                value: auto
+      - id: 102
+        name: fan_mode
+        type: string
+        mapping:
+          - dps_val: "smart"
+            value: auto
+          - dps_val: "silence"
+            value: quiet
+            icon: "mdi:fan-minus"
+          # this is false, but it's showing as an option
+          # - dps_val: "booster"
+          #   value: Boost
+          #   icon: "mdi:fan-plus"
+          #   available: boost_available
+      - id: 103
+        name: current_temperature
+        type: integer
+      - id: 106
+        name: mode
+        type: string
+        hidden: true
+      - id: 107
+        name: temperature
+        type: integer
+        unit: C
+        range:
+          min: 18 # dps 108 is target_temp_low - can this value be driven by that?
```

Comment:
min and max temperature will be passed to HA as the range if available, to constrain the UI. But the widest range supported should still be specified here as a backup, and will be used to prevent out of range values being sent to the device.